styles/variables-silicon.less

```diff
@@ -68,13 +68,15 @@
 
 // Item
 // ---------------------------------------
-@agate-item-font-size: 40px;
+@agate-item-font-size: 32px;
 @agate-item-font-weight: 500;
 @agate-item-label-font-size: 30px;
-@agate-item-side-outset: 32px;
-@agate-item-height: 90px;
+@agate-item-side-outset: 45px;
+@agate-item-top-bottom-outset: 26px;
+@agate-item-height: 42px;
 @agate-item-height-large: 120px;
-@agate-item-border-width: 3px;
+@agate-item-border-width: 0 0 2px 0;
```

Comment:
I think we'll need to set this value to `3px`, rather than `2px` because of the screen scaling and number rounding issues.